app/initial_state.js

```diff
@@ -293,9 +293,13 @@ const state = {
             options: [],
             visible: false
         },
+        root: {
```

Comment:
Actually we don't need this. I was keeping it around to reset the state, but we can just pass undefined to the baseReducer again which will reset us to the initialState. I'll make the change and if you approve then I'll remove the file.